src/pages/Login/index.tsx

```diff
@@ -85,89 +78,124 @@ const LoginPage: React.FC = () => {
+          to={{
+            pathname: FrontendRoute.CREATE_ACCOUNT_PAGE,
+            state: { email, password },
+          }}
+        >
+          <Button type="button" variant={ButtonVariant.Secondary}>
+            Register
+          </Button>
+        </Link>
+      </div>
+    </Form>
+  );
+
+  return (
+    <>
       <Helmet>
         <title>Login | {HACKATHON_NAME}</title>
       </Helmet>
-      <Form>
-        <Flex
-          alignItems={'center'}
-          flexDirection={'column'}
-          p={'4rem 0rem 0rem 5.8rem'}
+
+      <div className="login-form-container">
+        <H1
+          fontSize={'30px'}
+          textAlign={'left'}
+          marginTop={'0px'}
+          marginBottom={'20px'}
+          marginLeft={'0px'}
+          paddingBottom={'20px'}
+          paddingTop={'70px'}
         >
-          <Box alignSelf={'flex-start'}>
-            <H1 fontSize={'24px'} marginLeft={'0px'}>
-              Sign in / Register
-            </H1>
-          </Box>
-          <EmailInput
-            label={EMAIL_LABEL}
-            onEmailChanged={setEmail}
-            value={email}
-            isTight={true}
-          />
-          <PasswordInput
-            label={PASSWORD_LABEL}
-            onPasswordChanged={setPassword}
-            value={password}
-            isTight={true}
-          />
-          <Box alignSelf={'flex-end'} mb={'30px'} pr={'10px'}>
-            <ForgotPasswordLinkComponent />
-          </Box>
-          <Flex>
-            <Box pr={'5px'}>
-              <Button type="button" onClick={handleSubmit}>
-                Sign in
-              </Button>
-            </Box>
-            <Box pl={'5px'}>
-              <Link
-                to={{
-                  pathname: FrontendRoute.CREATE_ACCOUNT_PAGE,
-                  state: { email, password },
-                }}
-              >
-                <Button type="button" variant={ButtonVariant.Secondary}>
-                  Register
-                </Button>
-              </Link>
-            </Box>
-          </Flex>
-        </Flex>
-      </Form>
-    </MaxWidthBox>
-  );
+          Sign in / Register
+        </H1>
+        {renderLoginForm()}
+      </div>
 
-  return (
-    <MediaQuery minWidth={1224} width={'100%'}>
-      {(matches) =>
-        matches ? (
-          <LeftContainer>
-            {renderForm()}
-            <BackgroundImage
-              src={Coders}
-              top={'60px'}
-              right={'0px'}
-              imgWidth={'100%'}
-              imgHeight={'100%'}
-              minHeight={'600px'}
-            />
-          </LeftContainer>
-        ) : (
-          <div>
-            {renderForm()}
-            <BackgroundImage
-              src={Coders}
-              top={'60px'}
-              right={'0px'}
-              imgHeight={'100%'}
-            />
-          </div>
-        )
-      }
-    </MediaQuery>
+      <img className="background-image coders" src={Coders} />
+
+      <style jsx>{`
```

Comment:
This is moving away from the original organization of the codebase whereby all style elements would be in the shared element library. 

Is it standard to have style tags interspersed with the JSX here? 

Is it possible to refactor or abstract away the styling?

Reply:
Yeah, this PR represents an experiment for what styling could look like, and won't be merged until I've finished a design doc for how this refactor would apply across the board.

The overall idea is to remove css properties that are attached to tags such as 
```
<Flex
          alignItems={'center'}
          flexDirection={'column'}
          p={'4rem 0rem 0rem 5.8rem'}/>
```

or 

```
<H1
          fontSize={'30px'}
          textAlign={'left'}
          marginTop={'0px'}
          marginBottom={'20px'}
          marginLeft={'0px'}
          paddingBottom={'20px'}
          paddingTop={'70px'}>
```

and put them back into css, but scoped just to the component using [styled-jsx](https://github.com/vercel/styled-jsx).

This will make the styles more modular and make our JSX less cumbersome (e.g. the numerous MediaQueries and repetition of `<ManageAccountForm>` and `<H1>` in https://github.com/hackmcgill/dashboard/blob/develop/src/pages/Account/Create.tsx could all go away and become super readable)